dnf5/main.cpp

```diff
@@ -1444,6 +1444,11 @@ int main(int argc, char * argv[]) try {
 
     log_router.info("DNF5 finished");
 
+    // Print Complete! message only when transaction is created to prevent poluting output from repoquery or other command
+    if (auto * transaction = context.get_transaction(); transaction && !transaction->empty()) {
+        std::cout << "Complete!\n";
```

Comment:
I find the exclamation mark for an information message inappropriate. Could we use "Complete.\n" instead?

Reply:
I have no preferences. `"Complete!"` was used by DNF4.